WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
Oh, wait. The scrolling sems to work, actually. Maybe it's just my browser acting weird.

I have no idea about the floating, though.